eclair-core/src/main/scala/fr/acinq/eclair/Eclair.scala

```diff
@@ -225,13 +227,15 @@ class EclairImpl(appKit: Kit) extends Eclair {
     * @param request
     * @return
     */
-  def sendToChannel(channelIdentifier: String, request: Any)(implicit timeout: Timeout): Future[Any] =
-    for {
-      fwdReq <- Future(Register.ForwardShortId(ShortChannelId(channelIdentifier), request))
-        .recoverWith { case _ => Future(Register.Forward(ByteVector32.fromValidHex(channelIdentifier), request)) }
-        .recoverWith { case _ => Future.failed(new RuntimeException(s"invalid channel identifier '$channelIdentifier'")) }
-      res <- appKit.register ? fwdReq
-    } yield res
+  def sendToChannel(channelIdentifier: String, request: Any)(implicit timeout: Timeout): Future[Any] = {
```

Comment:
What is the purpose of this change?

Reply:
Purpose was so get rid of unnecessary "Future" and simplify the code but #969 got a better solution so i'm removing this.